pyiceberg/transforms.py

```diff
@@ -106,6 +109,17 @@
 TRUNCATE_PARSER = ParseNumberFromBrackets(TRUNCATE)
 
 
+def _try_import(module_name: str, extras_name: str | None = None) -> types.ModuleType:
```

Comment:
nit: feels like we should move this somewhere else, other than in `transforms.py` but i cant find a suitable place

Reply:
i think its fine to leave this here now :)